include/cute_draw.h

```diff
@@ -1178,6 +1178,18 @@ CF_API CF_RenderState CF_CALL cf_draw_peek_render_state();
  */
 CF_API void CF_CALL cf_draw_set_atlas_dimensions(int width_in_pixels, int height_in_pixels);
 
+/**
+ * @struct   CF_DrawShaderBytecode
+ * @category draw
+ * @brief    Bytecode for a draw shader.
+ * @remarks  This can be created using the `cute-shaderc` compiler.
+ * @related  CF_Shader cf_draw_push_shader cf_draw_pop_shader cf_draw_peek_shader cf_make_draw_shader_from_bytecode
+ */
+typedef struct CF_DrawShaderBytecode {
```

Comment:
Newline on brace to match the rest of CF. We will need a page on CF website documenting `cute-shaderc` for sure. I can help get you going on that if you need -- it's really easy.